commit 9465c594e919bd2ac3e41ac96e72a432daf81fa7
Author: Alan Coopersmith <alan.coopersmith@sun.com>
Date:   Tue Jan 9 19:33:45 2007 -0800

    Was previously done in monolith Imakefile as:
    SpecialCObjectRule(s3_video,$(_NOOP_), -DS3_NEWMMIO)
    
    Needed to get definitions of GET_PSTREAM_*, GET_SSTREAM_*, etc.

diff --git a/src/s3_video.c b/src/s3_video.c
index 77f4930..8189fdf 100644
--- a/src/s3_video.c
+++ b/src/s3_video.c
@@ -30,6 +30,8 @@ #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
 
+#define S3_NEWMMIO	/* previously defined in Imakefile in monolith */
+
 #include "xf86.h"
 #include "xf86_OSproc.h"
 
